includes/core-changes/aspnetcore/5.0/blazor-components-trim-insignificant-whitespace.md

```diff
@@ -0,0 +1,73 @@
+### Blazor: Insignificant whitespace trimmed from components at compile time
+
+Starting with ASP.NET Core 5.0 Preview 7, the Razor compiler omits insignificant whitespace in Blazor components (*.razor* files) at compile time. For discussion, see issue [dotnet/aspnetcore#23568](https://github.com/dotnet/aspnetcore/issues/23568).
+
+#### Version introduced
+
+5.0 Preview 7
+
+#### Old behavior
+
+In 3.x versions of Blazor Server and Blazor WebAssembly, *.razor* files preserved whitespace in the text context of the rendered output. Consider the following Blazor component code:
```

Comment:
Is "text context" correct and if so will the meaning be obvious to the intended audience?

Reply:
@SteveSandersonMS What's another way to describe "text context"?

Reply:
Maybe phrase as something like:

> In 3.x versions of Blazor Server and Blazor WebAssembly, *.razor* components preserved any whitespace found in the source code by rendering corresponding whitespace-only text nodes in the browser's DOM.